tensorflow/core/public/version.h

```diff
@@ -116,5 +116,7 @@ extern const char* tf_compiler_version();
 // The git commit designator when tensorflow was built
 // If no git repository, this will be "internal".
 extern const char* tf_git_version();
+// Value of the _GLIBCXX_USE_CXX11_ABI flag, or -1 if it's not set.
```

Comment:
Any reason it's not 0 when not set?

Reply:
Allows to skip it in `tf.sysconfig.get_compile_flag()` if it's not defined in the system.  If you'd rather it always be there and set to 0 by default, I can change.

Reply:
OK, sorry, now I get your point. Let's leave it to `-1`, then.